src/presentation/controllers/customer-controller.js

```diff
@@ -0,0 +1,26 @@
+import { validationResult } from 'express-validator'
+import Customer from '../../domain/user/model/user_model.js'
+
+const customerAddedList = [];
+class CustomerController {
+
+
+    createCustomer(req, res, next) {
+        const errors = validationResult(req)
+        if (!errors.isEmpty()) {
+            return res.status(422).json({ errors: errors.mapped() })
+        }
+        var newCustomer = new Customer(req.body);
+        customerAddedList.push(newCustomer);
+        console.log(customerAddedList);
```

Comment:
Opa não é bom commitar incluindo console.logs (por uma questão de performance já que eles são síncronos e também por que não é algo necessário em produção já que o que precisar ser loggado será através de uma classe logger própria), se der remova eles antes do commit :D

Reply:
conseole.log retirado 
```suggestion
        console.log(customerAddedList);
```